src/stretchy.js

```diff
@@ -107,7 +107,7 @@ const mathMLnode = function(label: string): mathMLTree.MathNode {
 // corresponds to 0.522 em inside the document.
 
 const katexImagesData: {
-    [string]: ([string[], number, number] | [string[], number, number, string])
+    [string]: ([string[], number, number] | [[string], number, number, string])
```

Comment:
Why did we change from an array of strings to a one-tuple in the second option?

Reply:
This is such a weird typing.  Hopefully we can making things a bit more regular in the future.  No action required.